client/src/routes/+layout.svelte

```diff
@@ -1,166 +1,12 @@
-        prompt.
-      </li>
-      <li>Cats must try to find any fishy answers that don’t make sense with the given icebreaker.</li>
-      <li>
-        Catfish, defend yourself. Make excuses, lie, and find any way to convince your partner that your answer fits
-        their icebreaker.
-      </li>
-      <li>
-        Have an odd number of players and couldn’t find a match? Good thing this is a loner-free zone! Take your
-        vengeance on another couple and stalk their date as they have a great time without you! Unaccompanied players
-        will get a new menu that will allow them to select any current pair and will be able to watch over their current
-        chat.
-      </li>
-    </ul>
-
-    <h3 class="mdc-typography--headline5">Voting</h3>
-    <ul class="mdc-typography--body1">
-      <li>
-        After (hopefully) all players have had their fun on their dates and everyone’s inner Purrlock Holmes has
-        activated, players will now be able to talk to each other to report what they have witnessed.
-      </li>
-      <li>During this phase, the answers to the icebreakers for every player will be shown.</li>
-
-      <li><strong>Cats: </strong>find any answer you find to be strange and call out your fishy prey.</li>
-      <li>
-        <strong>Catfish:</strong> your furry feline friends are on the hunt to remove you from the app. Defend your answer
-        amongst the group, call out other cats, or stay completely silent. Do whatever you can to survive the round and not
-        be voted out.
-      </li>
-      <li>
-        Once a player feels that they know who they want removed, they may click on the player to cast a vote on them.
-        Catfish, try to blend in and vote out any cats that are being backstabbed.
-      </li>
-      <li>
-        Once the timer has ended, the player with the most votes will be kicked from the current chat session and will
-        have to spectate the rest of the players.
-      </li>
-    </ul>
-
-    <h3 class="mdc-typography--headline5">End</h3>
-    <ul class="mdc-typography--body1">
-      <li>Every player is given one of two roles: <i>Cat</i> or <i>Catfish.</i></li>
-      <li>
-        The game will come to an end with one of two scenarios:
-        <ul>
-          <li>
-            <strong>Scenario 1: </strong>There are zero catfish remaining.
-          </li>
-          <li><strong>Scenario 2: </strong>The number of cats are equal to the number of catfish.</li>
-        </ul>
-      </li>
-      <li>Scenario 1 will result in the cats winning. The app is now fish-free!</li>
-      <li>Scenario 2 will result in the catfish winning and cats dropping lower in the food chain…</li>
-    </ul>
-  </Content>
-</Dialog>
-
-<div class="top-left">
-  <IconButton class="material-icons" on:click={() => (showRulesDialog = true)}>help</IconButton>
-</div>
+<header class="header">
+  <Rules />
+  <slot name="top-right" />
+</header>
```

Comment:
move this into a component and then use that component in each route

this is a workaround for https://github.com/sveltejs/kit/issues/627